tensorflow/tools/lib_package/BUILD

```diff
@@ -120,9 +119,8 @@ genrule(
         "@jpeg//:LICENSE.md",
         "@libxsmm_archive//:LICENSE",
         "@local_config_sycl//sycl:LICENSE.text",
-        "@nanopb_git//:LICENSE.txt",
```

Comment:
We need to include this license for the sanity checks to pass. You can run them yourself under `tensorflow/tools/ci_build/ci_sanity.sh`.

I just sent a change to gRPC so that the license file is exported so it can be used here:
https://github.com/grpc/grpc/pull/9711